### PR TITLE
Added my project code to 'homeworks' - HTML-CSS-Popup

### PR DESCRIPTION
**Pull request by Acerpo**
### HTML-CSS-Popup

[Demo](https://acerpo.github.io/HTML-CSS-Popup/) |
[Code base](https://github.com/Acerpo/HTML-CSS-Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/acerpo/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="ru">
```
Comment (A-Ostrovnyy) [inline submissions/acerpo/index.html:2]: The whole page uses the English language. but you set russuan language, set the correct value for lang attribute? ita' important for SEO and accessibilty

File: submissions/acerpo/index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style/style.css">
+
+  <title>HTML-CSS-Popup</title>
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <nav class="nav">
+                <div class="nav__item">
```
Comment (A-Ostrovnyy) [inline submissions/acerpo/index.html:14]: For nested navigation better to use an unordered list:
```HTML
<nav>
  <ul>
    <li>
      <a>Na link</a>
    </li>
     <li>
      <ul>
        <li>
          <a>Nav link</a>
        </li>
      </ul>
    </li>
  </ul>
</nav>
```

File: submissions/acerpo/index.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                            <a class="popup_content_body_link" href="#"><img src="./img/search.png" alt="Search">Search</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/photos.png" alt="Photo">Photo</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/apps.png" alt="Apps">Apps</a>
+                            <a class="popup_content_body_link" href="#"><img src="./img/documents.png" alt="Document">Document</a>
+                            <div class="more">
+                                <input class="checkbox_input checkbox_more" type="checkbox" id="checkbox_toggle_more">
+                                <label class="more_label" for="checkbox_toggle_more">More</label>
+                                <div class="more_content">
+                                    <div class="more_content_body">
+                                        <a class="popup_content_body_link" href="#"><img src="./img/chat.png" alt="Chat">Chat</a>
```
Comment (A-Ostrovnyy) [inline submissions/acerpo/index.html:42]: For icons better set an empty `alt` attribute.
Sorry for russian but it's a good [article](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/acerpo/index.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                </div>  
+                <div class="nav__item">
+                    <a class="headerLink" href="#"><img class=" user_img" src="./img/favicon.png" alt="user"></a>
+                </div>      
+            </nav>
+        </div>
+    </header>
+  
+</body>
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/acerpo/index.html:66]: Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

File: submissions/acerpo/style/style.css [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+.nav {
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+}
+
+.nav__item {
+padding: 0 10px;
```
Comment (A-Ostrovnyy) [inline submissions/acerpo/style/style.css:12]: Use [Prettier](https://prettier.io/) for autoformating

File: submissions/acerpo/style/style.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+.headerLink {
+    font-family: 'Segoe UI';
```
Comment (A-Ostrovnyy) [inline submissions/acerpo/style/style.css:16]: Always set a fallback font. [Reason](https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/)

File: submissions/acerpo/style/style.css [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+    width: 100%;
+}
+
+.user_img {
+    width: 28px;
+    padding: 3px;
+    margin-bottom: -3px;
+    border: 1px solid;
+    border-radius: 100%
+
```
Comment (A-Ostrovnyy) [inline submissions/acerpo/style/style.css:33]: Avoid unused lines inside CSS rules block

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below, also, if you want check how looks/works your popup in firefox)

File: submissions/acerpo/index.html [hunk 8]
```diff
@@ -11,56 +11,57 @@
...
+                            <ul class="manu-1">
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/calendar.png" alt="Calendar">Calendar</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/mail.png" alt="Mail">Mail</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/news.png" alt="News">News</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/maps.png" alt="Map">Map</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/files.png" alt="Files">Files</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/contacts.png" alt="Contacts">Contacts</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/tanslation.png" alt="Translate">Translate</a>
+                                <li><a class="popup_content_body_link" href="#"><img src="./img/videos.png" alt="Videos">Videos</a>
```
Comment (A-Ostrovnyy) [inline submissions/acerpo/index.html:33]: You forget about `alt`) [https://github.com/kottans/frontend-2022-homeworks/pull/26#discussion_r933176977](https://github.com/kottans/frontend-2022-homeworks/pull/26#discussion_r933176977)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One more issue needs to be done.

File: submissions/acerpo/index.html (hunk 8 above)
Comment (Acerpo) [reply submissions/acerpo/index.html:33]: Looks like I misunderstood the article you posted.

Why is it indicated in the task that you must not forget to specify the data in ALT?

Comment (A-Ostrovnyy) [reply submissions/acerpo/index.html]: We do not restrict your design, on previous courses students created a full-page search with a Google-like design. Also, if you add a logo you should fill `alt`.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**